src/pages/Group/Description.vue

```diff
@@ -18,7 +18,7 @@ export default {
   components: { QCard, Markdown },
   computed: {
     ...mapGetters({
-      group: 'groups/activeGroup',
+      group: 'currentGroup/get',
```

Comment:
`get` sounds like it would be a function-getter, but it's presumably not? maybe `group` is a better name? (even though it's kind of duplicated...)

Reply:
or something like `value`?